content/ru/docs/intro/using_helm.md

```diff
@@ -0,0 +1,482 @@
+```
+
+Из примера выше видно, что релиз `happy-panda` был удален.
+
+В предыдущих версиях Helm, когда релиз удалялся, запись о его
+удалении оставалась. В Helm 3, Удаление удаляет записи релиза, а также.
+Если вы хотите сохранить запись об удалении выпуска, используйте `helm uninstall
+--keep-history`. Использование `helm list --uninstalled` покажет только те выпуски, которые
+были удалены с флагом `--keep-history`.
+
+Флаг `helm list --all` покажет вам все релизы,
+сохраненные Helm, включая записи для неудачных или удаленных элементов (если был указан параметр `--keep-history`
+):
+
+```console
+$  helm list --all
+NAME            VERSION UPDATED                         STATUS          CHART
+happy-panda     2       Wed Sep 28 12:47:54 2016        UNINSTALLED     wordpress-10.4.5.6.0
+inky-cat        1       Wed Sep 28 12:59:46 2016        DEPLOYED        alpine-0.1.0
+kindred-angelf  2       Tue Sep 27 16:16:10 2016        UNINSTALLED     alpine-0.1.0
+```
+
+Обратите внимание, что поскольку релизы теперь удаляются по умолчанию, откат удаленного ресурса больше невозможен.
+
+## 'helm repo': Работа с Репозиториями
+
+Helm 3 больше не поставляется со стандартным хранилищем chart-ов. Команды `helm repo`
+предоставляет возможности добавления, перечисления и удаления репозиториев.
+
+Узнать о настроенных репозитории можно с помощью `helm repo list`:
+
+```console
+$ helm repo list
+NAME            URL
+stable          https://charts.helm.sh/stable
+mumoshu         https://mumoshu.github.io/charts
+```
+
+Новые репозитории могут быть добавлены с помощью `helm repo add`:
+
+```console
+$ helm repo add dev https://example.com/dev-charts
+```
+
+Поскольку репозитории диаграмм часто меняются, в любой момент вы можете убедиться,
+что ваш клиент Helm обновлен, запустив `helm repo update`.
+
+Репозитории могут быть удалены с помощью `helm repo remove`.
+
+## Создание Собственных Chart-ов
+
+[Руководство по разработке Chart-ов](https://helm.sh/docs/topics/charts/) объясняет, как
+создавать свои собственные chart-ы. Но вы можете быстро начать работу с помощью команды `helm
+create`:
+
+```console
+$ helm create deis-workflow
+Creating deis-workflow
+```
+
+Теперь у вас есть chart в `./deis-workflow`. Вы можете редактировать его и создавать свои собственные
+шаблоны.
+
+
+У Helm есть линтер для проверки корректности chart-а, воспользоваться им можно с помощью `helm
+lint`.
+
+Когда придет время упаковать chart в пакет для распространения вам поможет команда `helm
+package`
+
+```console
+$ helm package deis-workflow
+deis-workflow-0.1.0.tgz
+```
+
+Теперь вы можете с легкостью поставить этот chart в ваш Helm с помощью `helm install`
+
+```console
+$ helm install deis-workflow ./deis-workflow-0.1.0.tgz
+...
+```
+
+Упакованные chart-ы могут быть загружены в репозиторий chart-ов. Смотрите документацию по
+[Helm chart репозитории](https://helm.sh/docs/topics/charts/) для того, что бы узнать новые детали.
```

Comment:
How can I do"ref" without a direct link? The page with the Russian translation is not yet available

Reply:
You either can add a copy of the untranslated page under `ru` (this is okay) or follow the suggestions in https://helm.sh/docs/community/localization/#resolving-internal-links to change the links.